private/model/api/load.go

```diff
@@ -167,6 +167,11 @@ func (a *API) Setup() {
 	a.setMetadataEndpointsKey()
 	a.writeShapeNames()
 	a.resolveReferences()
+
+	if !a.NoRemoveUnusedShapes {
+		a.removeUnusedShapes()
+	}
+
 	a.fixStutterNames()
 	a.renameExportable()
 	a.applyShapeNameAliases()
```

Comment:
Curiously, are these indents intended or should they be aligned?

Reply:
Looks like this was just a visual bug with Github's diff view.